src/index/coinstatsindex.cpp

```diff
@@ -107,7 +107,7 @@ CoinStatsIndex::CoinStatsIndex(size_t n_cache_size, bool f_memory, bool f_wipe)
 bool CoinStatsIndex::WriteBlock(const CBlock& block, const CBlockIndex* pindex)
 {
     CBlockUndo block_undo;
-    const CAmount block_subsidy{GetBlockSubsidy(pindex->pprev->nBits, pindex->pprev->nHeight, Params().GetConsensus())};
+    const CAmount block_subsidy{pindex->nHeight > 0 ? GetBlockSubsidy(pindex->pprev->nBits, pindex->pprev->nHeight, Params().GetConsensus()) : Params().GenesisBlock().vtx[0]->GetValueOut()};
```

Comment:
dunno why you preferred this more complex version compare to previous implementation of same case in `rpc/blockchain.cpp`: https://github.com/dashpay/dash/blob/master/src/rpc/blockchain.cpp#L2240 

btw, duplicated code with line#380 (see below)

Reply:
Because I don't want to hardcode the subsidy value and the code will be deduplicated in a future PR